lib/commands/web.js

```diff
@@ -387,21 +387,23 @@ helpers.useAtomsElement = function useAtomsElement (el) {
 };
 
 helpers.convertElementsForAtoms = function convertElementsForAtoms (args = []) {
-  let newArgs = [];
-  for (let arg of args) {
+  const resultArgs = [];
+  for (const arg of args) {
     if (helpers.hasElementId(arg)) {
       // Get the element key from W3C or MJSONWP key
-      let elementId = helpers.getElementId(arg);
-      let atomsElement = this.getAtomsElement(elementId);
+      const elementId = helpers.getElementId(arg);
+      const atomsElement = this.getAtomsElement(elementId);
       if (atomsElement === null) {
         throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${elementId}'`);
       }
-      newArgs.push(atomsElement);
+      resultArgs.push(atomsElement);
+    } else if (_.isArray(arg)) {
+      resultArgs.push(this.convertElementsForAtoms(arg));
```

Comment:
This will result in a multidimensional array. Needs to be `resultArgs.push(...this.convertElementsForAtoms(arg));`

Reply:
I assume this is expected, so the original input data like `[[{'ELEMENT': 5000}, {'ELEMENT': 5001}]]` is not flattened.

Reply:
Good point. Never mind.